docs/source/traditional-ml/serving-multiple-models-with-pyfunc/notebooks/MME_Tutorial.ipynb

```diff
@@ -0,0 +1,863 @@
+       "      <th></th>\n",
+       "      <th>x1</th>\n",
+       "      <th>x2</th>\n",
+       "      <th>x3</th>\n",
+       "      <th>y</th>\n",
+       "      <th>dow</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2024-01-25 11:52:32.719166</th>\n",
+       "      <td>1.211154</td>\n",
+       "      <td>0.051818</td>\n",
+       "      <td>-0.046877</td>\n",
+       "      <td>0.245944</td>\n",
+       "      <td>3</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-01-26 11:52:32.719166</th>\n",
+       "      <td>-0.277421</td>\n",
+       "      <td>0.260553</td>\n",
+       "      <td>-0.436572</td>\n",
+       "      <td>-1.230580</td>\n",
+       "      <td>4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-01-27 11:52:32.719166</th>\n",
+       "      <td>1.280508</td>\n",
+       "      <td>-0.502804</td>\n",
+       "      <td>-0.518665</td>\n",
+       "      <td>-2.165463</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-01-28 11:52:32.719166</th>\n",
+       "      <td>1.371376</td>\n",
+       "      <td>-0.755066</td>\n",
+       "      <td>0.743234</td>\n",
+       "      <td>1.787510</td>\n",
+       "      <td>6</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024-01-29 11:52:32.719166</th>\n",
+       "      <td>0.972065</td>\n",
+       "      <td>-0.268332</td>\n",
+       "      <td>2.404934</td>\n",
+       "      <td>0.156428</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                  x1        x2        x3         y  dow\n",
+       "2024-01-25 11:52:32.719166  1.211154  0.051818 -0.046877  0.245944    3\n",
+       "2024-01-26 11:52:32.719166 -0.277421  0.260553 -0.436572 -1.230580    4\n",
+       "2024-01-27 11:52:32.719166  1.280508 -0.502804 -0.518665 -2.165463    5\n",
+       "2024-01-28 11:52:32.719166  1.371376 -0.755066  0.743234  1.787510    6\n",
+       "2024-01-29 11:52:32.719166  0.972065 -0.268332  2.404934  0.156428    0"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def create_weekly_dataset(n_dates, n_observations_per_date):\n",
+    "    rng = pd.date_range(start=\"today\", periods=n_dates, freq=\"D\")\n",
+    "    df = pd.DataFrame(\n",
+    "        np.random.randn(n_dates * n_observations_per_date, 4),\n",
+    "        columns=[\"x1\", \"x2\", \"x3\", \"y\"],\n",
+    "        index=np.tile(rng, n_observations_per_date),\n",
+    "    )\n",
+    "    df[\"dow\"] = df.index.dayofweek\n",
+    "    return df\n",
+    "\n",
+    "\n",
+    "DF = create_weekly_dataset(n_dates=30, n_observations_per_date=500)\n",
+    "print(DF.shape)\n",
+    "DF.head()"
```

Comment:
Could we use `df` instead of `DF` due to the convention of python variables naming